ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -415,7 +422,7 @@ objects:
         if [[ "${CLUSTER_TYPE}" == "aws" ]]; then
             cat > /tmp/artifacts/installer/install-config.yaml << EOF
         apiVersion: v1beta4
-        baseDomain: ${BASE_DOMAIN}
+        baseDomain: origin-ci-int-aws.dev.rhcloud.com
```

Comment:
This might break folks who want to run soak tests in a different CI account or some such?  Can we leave the template-scoped parameter, default it to empty, use this fallback value for AWS, and error if it's non-empty on Azure?

Reply:
> This might break folks who want to run soak tests in a different CI account or some such? 

do we have docs on how to do that? So that i can take a look at how its done.